### DISCUSSION TITLE
Household use

### DISCUSSION THREAD

**Discussion by Keeper-of-the-Keys** [firefly-iii/firefly-iii#5005] [Firefly III]
Hey this looks like a really cool project and I hope to check it out in the near future.

One question that is not 100% clear to me from here:
https://docs.firefly-iii.org/firefly-iii/faq/general/

> Is it multi-user?
> 
> Yes. For security reasons Firefly III opens up the registration form only for the first user, but anybody can register an account if you enable this under Administration > Configuration. Keep in mind that you won't be able to share your administration.
> 

Does this mean that I won't be able to share the same administration with my wife unless we commit a cardinal sin of security?

Thanks!

**Comment by JC5** [comment]
Indeed. I advise a password manager with a shared vault.

**Comment by Keeper-of-the-Keys** [comment]
Thanks for the speedy answer!

Is this some type of conceptual limitation that there should be only one bookkeeper in a family because it seems to apply in just about all the OSS book keeping projects I have found?

**Reply by JC5** [reply]
I wouldn't know, I only use Firefly III.

But I know that Firefly III was built on users and user ID's. Updating every single query and all of the tables is a lot of work and I'm in no rush.

**Reply by Ezwen** [reply]
Hi! Sorry to dig out this discussion, but I was wondering if being able to have multiple different users working collaboratively on the same booking project was planned in the future?

**Reply by JC5** [reply]
It is planned, yes!

**Reply by Ezwen** [reply]
Great to hear :) thank you for your work!

**Reply by ahatdude** [reply]
> It is planned, yes!

Hey,
Sorry to dig up this old thread but I assume it's better than creating a new one. Is this something that you think will happen soon? I just set up the server with accounts, budgets and piggy banks but it's a shame my SO needs to access via my account, since it just makes more sense to have her access everything via a non-admin account.

Thanks for your time.
